usb-pd/src/messages/pdo.rs

```diff
@@ -102,11 +141,25 @@ bitfield! {
         pub supply: u8 @ 28..=29,
         pub pps_power_limited: bool @ 27,
         /// Maximum voltage in 100mV increments
-        pub max_voltage: u8 @ 17..=24,
+        pub raw_max_voltage: u8 @ 17..=24,
         /// Minimum Voltage in 100mV increments
-        pub min_voltage: u8 @ 8..=15,
+        pub raw_min_voltage: u8 @ 8..=15,
         /// Maximum Current in 50mA increments
-        pub maximum_current: u8 @ 0..=6,
+        pub raw_max_current: u8 @ 0..=6,
+    }
+}
+
+impl SPRProgrammablePowerSupply {
+    pub fn max_voltage_mv(&self) -> u16 {
```

Comment:
I understand this is subjective, but I'm not loving the `_mv` here. Which of the following do you think is best?

* `max_voltage() -> u16`, with a doc comment explaining the return value is in millivolts
* `max_voltage() -> MilliVolts` where `MilliVolts` is some type representing voltage units (I'm assuming there must be some crate with these types already)
* `max_voltage_millivolts() -> u16`

Reply:
There is `uom` which is meant to have no_std functionality and has types for e.g. voltage; So i can have a look at that to see if that ends up being overkill or note.. 

fwiw I discounted the first option is it ends up being surprising if voltage doesn't give you volts.. The third  option is also ok by me, it's just a bit more typing (with mv being a common shorthand)

Reply:
I like `uom`, I'm happy to merge this commit as is, and I'll implement it.